### PR TITLE
qna authoring tests

### PR DESCRIPTION
**Pull request by mshaban-msft**

### CODE REVIEW

File: sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/test_create_and_deploy_project.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    GlobalQuestionAnsweringAccountPreparer,
+    QnaAuthoringHelper
+)
+
+from azure.ai.language.questionanswering.projects import QuestionAnsweringProjectsClient
+
+class CreateAndDeployTests(QuestionAnsweringTest):
+
+    @GlobalQuestionAnsweringAccountPreparer()
+    def test_create_project(self, qna_account, qna_key):
```
Comment (annatisch) [inline sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/test_create_and_deploy_project.py:22]: Will these tests be recorded?

File: sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/test_create_and_deploy_project.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        deployment_name = "production"
+        deployment_poller = client.begin_deploy_project(
+            project_name=project_name,
+            deployment_name=deployment_name
+        )
+        deployment_poller.result()
+
+        # assert
+        '''
+        deployments = client.list_deployments(
```
Comment (annatisch) [inline sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/test_create_and_deploy_project.py:56]: Are we waiting on a fix in uncomment this section?

File: sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/test_update_knowledge_sources.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    GlobalQuestionAnsweringAccountPreparer,
+    QnaAuthoringHelper
+)
+
+from azure.ai.language.questionanswering.projects import QuestionAnsweringProjectsClient
+
+class SourcesQnasSynonymsTests(QuestionAnsweringTest):
+
+    @GlobalQuestionAnsweringAccountPreparer()
+    def test_add_source(self, qna_account, qna_key):
```
Comment (annatisch) [inline sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/test_update_knowledge_sources.py:22]: Once we're happy that these are good to go, we could duplicate them in async?

File: sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/test_create_and_deploy_project.py (hunk 1 above)
Comment (mshaban-msft) [reply sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/test_create_and_deploy_project.py:22]: yes, was just waiting for your review
didn't wanna add unnecessary extra files until i resolve all comments and make sure all tests are passing

File: sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/test_update_knowledge_sources.py (hunk 3 above)
Comment (mshaban-msft) [reply sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/test_update_knowledge_sources.py:22]: should also create samples/tests for async client?

Comment (annatisch) [reply sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/test_update_knowledge_sources.py:22]: Yup!

**Review by annatisch — APPROVED** [review]

**Pull request merged**